Update 3D map view configuration (#4140)
qgis/QGIS-Documentation: source/docs/user_manual/introduction/qgis_gui.rst

To do so, click the |options| :sup:`Configure...` button at the top of
the 3D canvas panel to open the :guilabel:`3D configuration` window.

.. _figure_3dmap_config:

.. figure:: img/3dmapconfiguration.png
   :align: center

   The 3D Map Configuration dialog
   

In the 3D Configuration window there are various options to
fine-tune the 3D scene:

* Camera's :guilabel:`Field of view`: allowing to create panoramic scenes.
  Default value is 45\°.
* :guilabel:`Terrain`: Before diving into the details, it is worth noting that
  terrain in a 3D view is represented by a hierarchy of terrain tiles and as
  the camera moves closer to the terrain, existing tiles that do not have
  sufficient details are replaced by smaller tiles with more details.
  Each tile has mesh geometry derived from the elevation raster layer and
  texture from 2D map layers.

  * :guilabel:`Type`: It can be :guilabel:`Flat terrain`, :guilabel:`DEM
    (Raster Layer)` or :guilabel:`Online`.
  * :guilabel:`Elevation`: Raster layer to be used for generation of the
    terrain. This layer must contain a band that represents elevation.
  * :guilabel:`Vertical scale`: Scale factor for vertical axis.

>     Increasing the scale will exaggerate the terrain.

Reviewer: ```suggestion
    Increasing the scale will exaggerate the height of the landforms.
```

  * :guilabel:`Tile resolution`: How many samples from the terrain raster layer
    to use for each tile.

>     A value of 16px means that the geometry of each tile will be built

Reviewer: ```suggestion
    A value of 16px means that the geometry of each tile will consist of
```

>     from 16x16 elevation samples.

Reviewer: ```suggestion
    16x16 elevation samples.
```

Reviewer: ?

    Higher numbers create more detailed terrain tiles at the expense of
    increased rendering complexity.
  * :guilabel:`Skirt height`: Sometimes it is possible to see small cracks
    between tiles of the terrain.
    Raising this value will add vertical walls ("skirts") around terrain
    tiles to hide the cracks.

>   * :guilabel:`Map theme`: Allows to select the set of layers to display in the

Reviewer: ```suggestion
  * :guilabel:`Map theme`: Makes it possible to select the set of layers to display in the
```

Reviewer: "Allows" used this way seems incomplete to me.  Perhaps "Allows one" or "Allows you"?

    map view from predefined :ref:`map themes <map_themes>`

> * |unchecked| :guilabel:`Terrain shading`: Allows to choose how the terrain

Reviewer: ```suggestion
* |unchecked| :guilabel:`Terrain shading`: Makes it possible to choose how the terrain
```

Reviewer: "Allows" used this way seems incomplete to me.  Perhaps "Allows one" or "Allows you"?

  should be rendered:

>   * Shading disabled - the color of terrain is determined only from map texture

Reviewer: ```suggestion
  * Shading disabled - terrain color is determined only from map texture
```

>   * Shading enabled - the color of terrain is determined using Phong's shading

Reviewer: ```suggestion
  * Shading enabled - terrain color is determined using Phong's shading
```

>     model, taking into account map texture, terrain normal vector, scene

Reviewer: ```suggestion
    model, taking into account map texture, the terrain normal vector, scene
```

>     light(s) and terrain material's :guilabel:`Ambient` and :guilabel:`Specular`

Reviewer: ```suggestion
    light(s) and the terrain material's :guilabel:`Ambient` and :guilabel:`Specular`
```

    colors and :guilabel:`Shininess`

> * :guilabel:`Lights`: You can add up to eight point lights, each with particular

Reviewer: ```suggestion
* :guilabel:`Lights`: You can add up to eight point lights, each with a particular
```
